core-web/libs/portlets/dot-locales/portlet/src/lib/dot-locales-list/store/dot-locales-list.store.spec.ts

```diff
@@ -45,14 +88,41 @@ describe('DotLocalesListStore', () => {
             expect(viewModel.locales[1].language).toBe('Spanish - es');
             expect(viewModel.locales[1].country).toBe('Spain - ES');
             expect(viewModel.locales[1].defaultLanguage).toBe(false);
+            done();
         });
     });
 
-    it('should handle empty locales array', () => {
-        spectator.service.setLocales([]);
+    it('should make default locale', () => {
+        const mockDefaultLocaleId = 1;
+        spectator.service.makeDefaultLocale(mockDefaultLocaleId);
+
+        expect(languageService.makeDefault).toHaveBeenCalledWith(mockDefaultLocaleId);
+        expect(messageService.add).toHaveBeenCalled();
+    });
+
+    it('should delete locale', () => {
+        const mockLocaleId = 1;
+        spectator.service.deleteLocale(mockLocaleId);
+
+        expect(languageService.delete).toHaveBeenCalledWith(mockLocaleId);
+        expect(messageService.add).toHaveBeenCalled();
+    });
+
+    it('should set the local actions correctly', (done) => {
+        spectator.service.setLocales([...mockLocales]);
 
         spectator.service.vm$.subscribe((viewModel) => {
-            expect(viewModel.locales.length).toBe(0);
+            const defaultLocale = viewModel.locales[0];
+            const notDefaultLocale = viewModel.locales[1];
```

Comment:
check the lengths too so if we add more actions this test will fail

Reply:
good catch. ! done